lib/galaxy/dependencies/pinned-requirements.txt

```diff
@@ -52,7 +52,7 @@ cryptography==41.0.2 ; python_version >= "3.7" and python_version < "3.12"
 cwl-upgrader==1.2.8 ; python_version >= "3.7" and python_version < "3.12"
 cwl-utils==0.28 ; python_version >= "3.7" and python_version < "3.12"
 cwltool==3.1.20221109155812 ; python_version >= "3.7" and python_version < "3.12"
-cython==0.29.36 ; python_version >= "3.7" and python_version < "3.12"
+cython==3.0.0 ; python_version >= "3.7" and python_version < "3.12"
```

Comment:
Did you check if our dependencies (or at least the subset that doesn't have upstream wheels) still compile, e.g. with `--no-binary :all:` ?

Reply:
No, I haven't. Cython shouldn't be here at all, it's still dragged in by pysam. I hope https://github.com/pysam-developers/pysam/pull/1186 is merged sooner or later.